README.md

```diff
@@ -9,8 +9,9 @@ Note: docker registry must be [v2](https://docs.docker.com/registry/spec/api/).
 * `repository`: *Required.* The name of the repository, e.g.
 `concourse/docker-image-resource`.
 
-  Note: When configuring a private registry, you must include the port
-  (e.g. :443 or :5000) even though the docker CLI does not require it.
+  Note: When configuring a private registry **with self-signed certificate**,
```

Comment:
I think this is too specific - it could be using their own internal CA, not just self-signed. :slightly_smiling_face:

Reply:
I'm trying to think of a sentence that does make sense. How about `with an "invalid" certificate`?

Reply:
Maybe `when configuring a private registry using a non-root CA`? A lot of organisations have their own CA for internal traffic so I wouldn't want to say it's invalid.

Reply:
```suggestion
  Note: When configuring a private registry **using non-root CA**,
```

Sounds good!